service/create/policy.go

```diff
@@ -77,14 +82,18 @@ func createRole(svc *iam.IAM, kmsKeyArn, s3Bucket, clusterID string) error {
 	return nil
 }
 
-func createInstanceProfile(svc *iam.IAM, clusterID string) (string, error) {
+func (s *Service) createInstanceProfile(svc *iam.IAM, clusterID string) (string, error) {
 	clusterRoleName := fmt.Sprintf("%s-%s", clusterID, RoleNameTemplate)
 	clusterProfileName := fmt.Sprintf("%s-%s", clusterID, ProfileNameTemplate)
 
 	if _, err := svc.CreateInstanceProfile(&iam.CreateInstanceProfileInput{
 		InstanceProfileName: aws.String(clusterProfileName),
 	}); err != nil {
-		return "", err
+		if aerr, ok := err.(awserr.Error); ok && aerr.Code() == iam.ErrCodeEntityAlreadyExistsException {
```

Comment:
I think it would make sense to have a `IsEC2FooAlreadyExists(error) bool` function.